CMakeModules/AosCoreCommon.cmake

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (C) 2024 Renesas Electronics Corporation.
+# Copyright (C) 2024 EPAM Systems, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+include(ExternalProject)
+
+set(aoscorecommon_build_dir ${CMAKE_CURRENT_BINARY_DIR}/aoscorecommon)
+
+ExternalProject_Add(
+    aoscorecommon
+    PREFIX ${aoscorecommon_build_dir}
+    GIT_REPOSITORY https://github.com/MykolaSolyanko/aos_core_common_cpp.git
```

Comment:
Probably should be changed to cloud repo.

Reply:
Done